### ISSUE TITLE
Add 'magit' command alias for 'magit-status'.

### ISSUE THREAD

**Issue by rswgnu** [magit/magit#3290]
Often new users will install a package and want to explore it quickly before reading the manual.
For magit, with its huge number of interactive commands, M-x magit RET is presently of no help to get started.  Why not alias magit to magit-status and then a user could invoke it easily and naturally in any directory and then use ? to see what commands are available.

**Comment by vermiculus** [comment]
Alternatively, such a function could open up a 'getting started' info section or something similar.

**Comment by rswgnu** [comment]
On Thu, Nov 30, 2017 at 9:34 AM, Sean Allred <notifications@github.com>
wrote:

> Alternatively, such a function could open up a 'getting started' info
> section or something similar.
>
​Better would be to invoke magit-status since this will be used more often
then Info and display a message on how to get help (when the command is
invoked under the name 'magit').
​

**Comment by vermiculus** [comment]
Reasonable, but I wouldn't want to short the other entry points for magit. Maybe something like
```
M-x magit RET ; completing read on the following
What would you like to do?
- View the git status of this directory (M-x magit-status)
- [other entry points...]
```

**Comment by tarsius** [comment]
:scream:

I am currently working on a setup wizard and a manual for `ghub`. The latter currently also comes with an epilogue, which I will probably remove. To avoid it going to waste I will just leave it here:

> This is quite an extensive manual for a library that provides
> essentially a single entry-point (and a few tiny wrappers).
> 
> I have written it (as well as the setup wizard) because experience has
> shown that a significant subset of users has difficulties configuring
> such things on their own, either because they are new to the game (and
> should be welcomed warmly), or because they are a bit lazy.
> 
> This manual tries to answer all their questions without the author of
> Ghub, or the authors of the packages that use it, having to answer the
> same questions over and over again.  Some users will still not read it
> before asking for help, which, thanks to this effort, you can often do
> by giving them a link.
> 
> Writing this was a big upfront investment but I hope that it pays out
> in the long run.  Once Magit uses Ghub, tens of thousands of users
> will start using Ghub too, and even if only 1% of them need help, that
> still amounts to a lot of support requests.
> 
> PS: If you enjoy a bit of sarcastic commentary in documentation, then
> I can warmly recommend the auth-source manual.  Otherwise just try to
> ignore such things, package maintainers are just humans too, and after
> investing days to write documentation that is then being ignored, some
> of us occasionally resort to such remarks in order to unwind.

I had my share of spoon-feeding users for a while. Documentation-resistance isn't something that we should facilitate. I guess what is being proposed here (both variants) could be categorized as "being nice to newcomers". But "give a man a fish and you feed him for a day..." applies as well.

**Comment by tarsius** [comment]
I'll probably regret my previous post soon, but as I said maintainers are just humans too.

Ps: I tend to edit my comments on Github after the initial post. If you read them in an email client, then you don't see the edits.

**Comment by rswgnu** [comment]
On Thu, Nov 30, 2017 at 10:15 AM, Jonas Bernoulli <notifications@github.com>
wrote:

> I had my share of spoon-feeding users for a while.
> Documentation-resistance isn't something that we should facilitate. I guess
> what is being proposed here (both variants) could be categorized as "being
> nice to newcomers". But "give a man a fish and you feed him for a day..."
> applies as well.
>
​I read and write a lot of documentation, so I am not averse to it all, but
I think we all find that most users can consume it in only small bits
(hence the nodes in Info documentation).  There are so many layers of
tools, conventions, libraries and apis to software development today that
people have to get in and get out of toolsets quickly.  Emacs package
management now allows for this in Emacs and an experienced Emacs and git
user might have a reasonable expectation of being able to explore basics of
magit after reading just a short section of the magit manual or running
through a tutorial.  Key to this are intuitive ways to both start and quit
from magit, say from dired within a git repository.

Magit is great but there is room for improvement.
​

**Comment by tarsius** [comment]
I am taking of for the rest of the day. I don't feel constructive or open to suggestions today.

**Comment by rswgnu** [comment]
On Thu, Nov 30, 2017 at 10:53 AM, Jonas Bernoulli <notifications@github.com>
wrote:

> Closed #3290 <https://github.com/magit/magit/issues/3290> via 4a38ff8
> <https://github.com/magit/magit/commit/4a38ff8653f22c83dc69c769dfd357f776f2cb3b>
> .
>

​Great, thanks.

The doc string led me to​ look at the Getting Started node of the Magit
Manual again.  In it you refer to "this short tutorial" but don't define
that, so the reader doesn't know whether you are referring to the whole
manual or specific sections.  It would be helpful to clarify that.

Thanks for Magit and enjoy your free time.
​
